drivers/net/nsos_sockets.c

```diff
@@ -1591,3 +1593,108 @@ NET_DEVICE_OFFLOAD_INIT(nsos_socket, "nsos_socket",
 			NULL,
 			NULL, NULL,
 			0, &nsos_iface_offload_api, NET_ETH_MTU);
+
+#ifdef CONFIG_NET_NATIVE_OFFLOADED_SOCKETS_CONNECTIVITY_SIM
+
+struct nsos_conn_data {
+	struct k_work_delayable work;
+	struct net_if *iface;
+	k_timeout_t connect_delay;
+};
+
+#define NSOS_NET_IF_CTX_TYPE struct nsos_conn_data
+
+static void nsos_delayed_connect_fn(struct k_work *work)
+{
+	struct k_work_delayable *dwork = k_work_delayable_from_work(work);
+	struct nsos_conn_data *data = CONTAINER_OF(dwork, struct nsos_conn_data, work);
+
+	LOG_INF("NSOS: active");
+	net_if_dormant_off(data->iface);
+}
+
+static void nsos_net_if_init(struct conn_mgr_conn_binding *binding)
+{
+	struct nsos_conn_data *data = binding->ctx;
+
+	LOG_DBG("");
```

Comment:
Empty debug?

Reply:
It appears in the logging output with the function name, its commonly used in many drivers when you just want to log that the function is called.